internal/display/display.go

```diff
@@ -106,6 +108,15 @@ func (r *Renderer) Results(data []View) {
 			list = append(list, item.Object())
 		}
 		r.JSONResult(list)
+	case OutputFormatCSV:
+		rows := make([][]string, 0, len(data))
+		for _, d := range data {
+			rows = append(rows, d.AsTableRow())
+		}
+		if err := writeCSV(r.ResultWriter, data[0].AsTableHeader(), rows); err != nil {
+			r.Errorf("couldn't render results as csv: %v", err)
+			return
+		}
```

Comment:
**Request:** Can you provide a sample output of this perhaps?

Reply:
As an example, using the users search `auth0 users search -q "" --csv`, results in:

```
UserID,Email,Connection
auth0|1234567890,user1@example.com,Username-Password-Authentication
auth0|0987654321,user2@example.com,Username-Password-Authentication
```

And so forth.